ADQL.tex

```diff
@@ -1490,34 +1490,38 @@ \subsubsection{Datatype functions}
 and \verb:<value_expression_primary>: enables the use of geometric functions
 and column references.
 
-\subsubsection{Coordsys}
+\subsubsection{Coordinate system}
 \label{sec:geom.coordsys.param}
 
 For historical reasons, the geometry constructors (\verb:BOX:, \verb:CIRCLE:,
 \verb:POINT: and \verb:POLYGON:) all accept a string literal as the first
-argument, hereafter called the COOSYS argument. Since version 2.1 of the
-specification, this parameter has been made optional and marked as deprecated.
-Future versions of this specification may remove this parameter from the listed
-functions.
-
-This specification makes no guarantees about any semantics for the deprecated
-COOSYS argument. Queries against services implementing ADQL 2.1 (or higher)
-SHOULD omit it. For interoperability, queries against 2.0 services SHOULD NOT
-pass arguments with differing COOSYS arguments to \verb:CONTAINS: or
-\verb:INTERSECTS:, as behaviour is undefined in that case.
+argument, hereafter called the COOSYS argument.
 
 The COOSYS argument was originally intended to carry information on
 a reference system or other coordinate system metadata. This was helpful in
 order to deal with data specified in different coordinate systems while
 performing geometric operations. It was up to the ADQL service to perform
-the appropriate conversion to make these operations possible. Now this argument
-is deprecated and will later be removed, the users should explicitly ask for
-such conversion. If this behavior is desired, the ADQL implementers have to
-allow it through User Defined Functions. Then, it is recommended to define them
-as declared in the \CatalogueUDF{}.
+the appropriate conversion to make these operations possible.
+
+Since version 2.1, this argument is deprecated and has been made optional.
+Future versions of this specification will remove this parameter from the listed
+functions.
+
+Coordinate conversions SHOULD now be explictly asked. The ADQL implementers have
+to allow it through User Defined Functions. Then, it is recommended to define
```

Comment:
Perhaps we should mention the UDF catalogue more like "an interoperable facility for frame transformations is in preparation as of this writing and is expected to be part of the UDF catalogue".

Reply:
Done